.github/workflows/ci.yml

```
@@ -24,8 +24,19 @@ jobs:
       - name: Run Biome
         run: biome ci
 
-  build-and-test:
+  shellcheck:
+    name: Shell Script Linting
     needs: code-quality
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+      - name: Install ShellCheck
+        run: sudo apt-get install shellcheck
```

```suggestion
        run: sudo apt-get install shellcheck
```
Does it need sudo? scary

---

Apparently yes - https://github.com/herodevs/cli/actions/runs/14537990328/job/40790193078